scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.12.1] - 2018-03-12
+### Added
+- Added resource, job category, and other caching @scrosby
```

Comment:
I'd make this more general to be something like "Optimizations to schedule time"